src/components/grouped-character/grouped-character.test.js

```diff
@@ -0,0 +1,224 @@
+import * as React from "react";
+import { mount } from "@cypress/react";
+import GroupedCharacter from "./grouped-character.component";
+
+import {
+  fieldHelpPreview,
+  getDataElementByValue,
+  tooltipPreview,
+  commonDataElementInputPreview,
+} from "../../../cypress/locators/index";
+
+const specialCharacters = ["mp150ú¿¡üßä", "!@#$%^*()_+-=~[];:.,?{}&\"'<>"];
+
+const GroupedCharacterComponent = ({ onChange, ...props }) => {
+  const [state, setState] = React.useState("");
+
+  const setValue = ({ target }) => {
+    setState(target.value.rawValue);
+    if (onChange) {
+      onChange(target);
+    }
+  };
+
+  return (
+    <>
```

Comment:
There's no need for this to be wrapped in a React Fragment 👍🏻

Reply:
you are right, removed